src/server/src/main/java/io/cassandrareaper/service/RepairRunner.java

```diff
@@ -481,4 +501,18 @@ public void killAndCleanupRunner() {
     }
     Thread.currentThread().interrupt();
   }
+
+  private String metricName(String metric, String keyspace) {
+    String cleanClusterName = clusterName.replaceAll("[^A-Za-z0-9]", "");
+    String cleanKeyspaceName = keyspace.replaceAll("[^A-Za-z0-9]", "");
+    return MetricRegistry.name(RepairRunner.class, metric, cleanClusterName, cleanKeyspaceName);
+  }
+
+  private void updateRepairProgressMetric(int segmentsDone, int segmentsTotal) {
+    context.metricRegistry.remove(metricNameForRepairProgress);
+    context.metricRegistry.register(
+        metricNameForRepairProgress,
+        (Gauge<Float>) () -> (float) segmentsDone / segmentsTotal);
```

Comment:
this is a constant gauge. it feels a bit clumsy to bump it this way.

why not make `segmentsDone` a field, so the gauge does not need to be removed and re-registered again and again during the one repair…